rpcs3/Gui/GSFrame.cpp

```diff
@@ -22,6 +22,7 @@ GSFrame::GSFrame(const wxString& title) : wxFrame(nullptr, wxID_ANY, "GSFrame["
 	SetClientSize(res.width, res.height);
 	wxGetApp().Bind(wxEVT_KEY_DOWN, &GSFrame::OnKeyDown, this);
 	Bind(wxEVT_CLOSE_WINDOW, &GSFrame::OnClose, this);
+	Bind(wxEVT_LEFT_DCLICK, &GSFrame::OnLeftDclick, this);
```

Comment:
@Nekotekina 
Forgiveness, alone?
Had not https://github.com/RPCS3/rpcs3/pull/1242/files
Would more inteligant Merge my PR and then to the one at the scene to take me commit ...